apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/utils/contains-missing-block.js

```diff
@@ -0,0 +1,28 @@
+// Once parsed, missing Blocks have a name prop of `core/missing`.
+// see: https://github.com/WordPress/gutenberg/tree/742dbf2ef0e37481a3c14c29f3688aa0cd3cf887/packages/block-library/src/missing
+const MISSING_BLOCK_NAME = 'core/missing';
+
+/**
+ * Determines whether the provided collection of Blocks contains any "missing"
+ * blocks as determined by the presence of the `core/missing` block type.
+ *
+ * @param {Array} blocks the collection of block objects to check for "missing" block .
+ * @returns {boolean} whether the collection blocks contains any missing blocks.
+ */
+function containsMissingBlock( blocks ) {
+	return blocks.find( block => {
```

Comment:
minor: could you cast the returning value, adding `!!` of something like that.
It supposes to be a boolean, however, it's either a block object or `undefined`
Or update the jsdoc. 

```es6
return !! blocks.find( block => {
  // ...
}
```

Reply:
I agree. It's better to be explicit about the type here.

Reply:
Fixed.